### PR TITLE
add local: zh-cn

### PR DESCRIPTION
**Pull request by yorkie**
Add zh-cn version for this website.

### CODE REVIEW

**Comment by stevemao** [comment]
@nodejs/nodejs-cn

File: locale/zh-cn/site.json [hunk 1]
```diff
@@ -0,0 +1,169 @@
+{
+    "title": "Node.js",
+    "author": "Node.js 基金会",
+    "url": "https://nodejs.org/zh-cn/",
+    "locale": "zh-cn",
+    "scrollToTop": "置顶",
```
Comment (laosb) [inline locale/zh-cn/site.json]: “回到顶部”“回到页顶”

File: locale/zh-cn/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  lts: LTS
+  tagline-current: 最近更新
+  tagline-lts: 推荐
+  changelog: 更新日志
+  api: API 文档
+  version-schedule-prompt: 可参考
+  version-schedule-prompt-link-text: LTS 日程
+---
+
+Node.js®是一个基于[Chrome V8引擎](https://developers.google.com/v8/)的 JavaScript 运行时。
```
Comment (laosb) [inline locale/zh-cn/index.md]: `Chrome V8 引擎`，缺了个空格

File: locale/zh-cn/site.json [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+            "link": "download/package-manager",
+            "text": "下载／包管理器"
+        }
+    },
+    "docs": {
+        "link": "docs",
+        "text": "文档",
+        "es6": {
+            "link": "docs/es6",
+            "text": "ES6相关"
```
Comment (laosb) [inline locale/zh-cn/site.json]: “6”与 “ES” 间缺空格

File: locale/zh-cn/site.json [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+            "link": "foundation/tsc",
+            "text": "技术指导委员会"
+        },
+        "itn": {
+             "link": "foundation/in-the-news",
+             "text": "新闻"
+        },
+        "announce": {
+            "link": "foundation/announcements",
+            "text": "声明"
```
Comment (laosb) [inline locale/zh-cn/site.json]: 也许“公告”更好些？

File: locale/zh-cn/index.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+Node.js使用高效、轻量级的事件驱动、非阻塞 I/O 模型。Node.js 之生态系统是目前最大的开源包管理系统。
```
Comment (laosb) [inline locale/zh-cn/index.md]: Node.js 和“使用”间缺空格

File: locale/zh-cn/site.json (hunk 3 above)
Comment (yorkie) [inline locale/zh-cn/site.json]: ES6 之间加空格会不会有一些奇怪～

Comment (laosb) [inline locale/zh-cn/site.json]: @yorkie 喔不小心搞错了，应该是 ES6 和 “相关” 之间缺空格。

**Comment by yorkie** [comment]
Fixed, thank you @laosb

**Comment by leizongmin** [comment]
Good

**Comment by ghost** [comment]
this has i18n +1 and checks have passed, merging

**Comment by ghost** [comment]
thanks @yorkie, @laosb and @leizongmin!

**Comment by zanjs** [comment]
He looks great! 

Can I join you.

**Comment by fhemberger** [comment]
@zanjs The Chinese localization team's repository is here: https://github.com/nodejs/nodejs-zh-CN.
It's a good place to start.

**Pull request merged**